### PR TITLE
[webHosting] Add monitoring charts into webhosting part

### PR DESCRIPTION
**Pull request by VincentCasse**
Add the monitoring charts into the full descriptive page. It can help customer to know if they have an issue on their website.

### CODE REVIEW

File: app/pages/products/hosting-web/hosting-web.ts [hunk 1]
```diff
@@ -25,14 +31,30 @@ export class HostingWebPage extends ProductCore {
...
+    this.getChart(this.monitoring, this.monitoringPeriod);
+  }
+
+  getChart(type: string, period: string) {
+    this.loading.monitoring = true;
+    this.hostingWebService.getChart(this.serviceName, type, period)
+      .subscribe(
+        (stats) => this.stats = stats,
+        (err) => this.toast.error('Une erreur est survenue lors du chargement des statistiques : ' + JSON.parse(err._body).message).present(),
+        () => this.loading.monitoring = false
```
Comment (bnjjj) [inline app/pages/products/hosting-web/hosting-web.ts]: could you use ` .finally(() => this.loading.monitoring = false) ` instead the third argument please ? :)

File: app/pages/products/hosting-web/hosting-web.service.ts [hunk 2]
```diff
@@ -33,6 +33,54 @@ export class HostingWebService {
...
   }
 
+  getChart(serviceName: string, type: string, period: string) {
+    return this.ovhRequest.get(`/hosting/web/${serviceName}/statistics`, {
+      search: {
+        type,
+        period
+      }
+    }).map((stats) => {
+      let charts = stats.map( (serie) => {
```
Comment (bnjjj) [inline app/pages/products/hosting-web/hosting-web.service.ts]: useless space between ` .map( (serie) `
should be ` .map((serie) `

**Comment by VincentCasse** [comment]
Changes applies :)

File: app/pages/products/hosting-web/hosting-web.ts [hunk 3]
```diff
@@ -25,14 +31,30 @@ export class HostingWebPage extends ProductCore {
...
         }
       );
+
+    this.getChart(this.monitoring, this.monitoringPeriod);
+  }
+
+  getChart(type: string, period: string) {
+    this.loading.monitoring = true;
+    this.hostingWebService.getChart(this.serviceName, type, period)
+    .finally(() -> this.loading.monitoring = false )
```
Comment (bnjjj) [inline app/pages/products/hosting-web/hosting-web.ts]: Oups erreur c'est => et pas -> :p oublie pas d'indenter correctement stp

**Pull request merged**